app/controllers/items_controller.rb

```diff
@@ -11,10 +12,37 @@ def create
     redirect_to current_user
   end
 
+  def destroy
+    @item = current_user.items.find(params[:id])
+    @item.update_attribute(:completed_at, Time.now)
```

Comment:
So i think you have two competing concepts going on here. Completing vs destroying. In this particular app, the requirements are to "complete" an item by destroying it. We don't necessarily keep a record of those who were completed. And even here, you are marking it as complete, then destroying it anyway.